### PR TITLE
Belongs to Wondex upgrades

### PR DESCRIPTION
**Pull request by 5of9**
Added GSM and Devicestatus for Wondex Protocol

### CODE REVIEW

**Comment by tananaev** [comment]
See other pull request.

**Comment by 5of9** [comment]
Fixed

**Comment by tananaev** [comment]
Same problem as in the other pull request.

**Comment by 5of9** [comment]
again

File: web/l10n/en.json [hunk 1]
```diff
@@ -248,6 +248,8 @@
     "commandPhone": "Phone Number",
     "commandMessage": "Message",
     "commandTimezone": "Timezone Offset",
+    "commandgetModemStatus": "GSM Status",
```
Comment (tananaev) [inline web/l10n/en.json]: I think the text should be Modem status as well, not GSM.

**Comment by 5of9** [comment]
You're right. Could be also a 3G or 4G device. Changed

**Comment by tananaev** [comment]
Merged, thanks.

**Pull request merged**